compiler/optimizer/LocalOpts.cpp

```diff
@@ -3548,6 +3548,27 @@ int32_t TR_EliminateRedundantGotos::process(TR::TreeTop *startTree, TR::TreeTop
             !performTransformation(comp(), "%sEliminating goto at the end of block_%d with BBStart %p\n", optDetailString(), block->getNumber(), block->getEntry()->getNode()))
         continue;
 
+      // Blocks could be removed when the exception edges are removed from the block
+      // The IN edges saved in fixablePreds need to be updated
+      for (auto previousEdge = fixablePreds.before_begin(), currentEdge = fixablePreds.begin();
+           currentEdge != fixablePreds.end();
+           ++previousEdge, ++currentEdge)
+         {
+         TR::CFGEdge* current = *currentEdge;
```

Comment:
Please move the space and make this `TR::CFGEdge *current` to match the other declarations next to this one